package/MDAnalysis/analysis/rdf.py

```diff
@@ -342,31 +434,35 @@ def _conclude(self):
 
         for i, (ag1, ag2) in enumerate(self.ags):
             # Number of each selection
-            nA = len(ag1)
-            nB = len(ag2)
-            N = nA * nB
             indices.append([ag1.indices, ag2.indices])
 
             # Average number density
             box_vol = self.volume / self.n_frames
-            density = N / box_vol
+            density = 1 / box_vol
 
             if self._density:
-                rdf.append(self.count[i] / (density * vol * self.n_frames))
-            else:
                 rdf.append(self.count[i] / (vol * self.n_frames))
+            else:
+                rdf.append(self.count[i] / (density * vol * self.n_frames))
 
         self.rdf = rdf
         self.indices = indices
 
     def get_cdf(self):
-        """Calculate the cumulative distribution functions (CDF) for all sites.
-        Note that this is the actual count within a given radius, i.e.,
-        :math:`N(r)`.
+        r"""Calculate the cumulative counts for all sites.
+
+        This is the :ref:`cumulative count<equation-countab>` within a given
+        radius, i.e., :math:`N_{ab}(r)`.
```

Comment:
Why are we calculating the counts and calling it CDF? This is confusing... 

I made it clear in the text but still: why did we do this instead of having separate methods that just do the thing that they are named after?

Reply:
Yes, we should have seperate functions to give CDF and cummulative N.

Reply:
Ok, new issue.

Reply:
@VOD555 can you raise issues for MDAnalysis and PMDA, please?